examples/persistent_subscriptions.rs

```diff
@@ -0,0 +1,81 @@
+use eventstore::{
+    Client, PersistentSubscriptionOptions, PersistentSubscriptionToAllOptions, SubEvent,
+    SubscriptionFilter,
+};
+
+async fn create_persistent_subscription(client: &Client) -> eventstore::Result<()> {
+    // #region create-persistent-subscription-to-stream
+    client
+        .create_persistent_subscription("test-stream", "subscription-group", &Default::default())
+        .await?;
+    // #endregion create-persistent-subscription-to-stream
+
+    Ok(())
+}
+
+async fn connect_to_persistent_subscription_to_stream(client: &Client) -> eventstore::Result<()> {
+    // #region subscribe-to-persistent-subscription-to-stream
+    let (mut read, mut write) = client
+        .connect_persistent_subscription("test-stream", "subscription-group", &Default::default())
+        .await?;
+
+    while let Some(event) = read.try_next_event().await? {
+        // Doing some productive work with the event...
+        let _ = write.ack_event(event).await;
```

Comment:
We have a discrepency in the informations we get back in this method between at leas .net & rust (RFC022 needs some addedum to make it clear ) 
.net give the Subscription client, retry count & resolved event.

Reply:
Done